sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs

```diff
@@ -765,17 +767,20 @@ async Task SetBreakpoint (SessionId sessionId, DebugStore store, BreakpointReque
 				return;
 			}
 
-			var locations = store.FindBreakpointLocations (req).ToList ();
+			var comparer = new SourceLocation.LocationComparer ();
+			// if column is specified the frontend wants the exact matches
+			// and will clear the bp if it isn't close enough
+			var locations = store.FindBreakpointLocations (req)
+				.Distinct (comparer)
+				.Where (l => l.Line == req.Line && (req.Column == 0 || l.Column == req.Column))
```

Comment:
Is this with 1-based column numbers?

Reply:
no, SourceLocation and BreakpointRequest are zero based like cdp

```
public SourceLocation (MethodInfo mi, SequencePoint sp)
		{
			this.id = mi.SourceId;
			this.line = sp.StartLine - 1;
			this.column = sp.StartColumn - 1;
			this.cliLoc = new CliLocation (mi, sp.Offset);
		}
```

Reply:
Then how do we differentiate between requests for exactly `column:0`, and inexact column (all breakpoints matching the line)?

Reply:
it doesn't matter the point of the patch is that column:0 == first bp on line whatever the column.